cgi-bin/DW/Controller/RPC/CutExpander.pm

```diff
@@ -99,7 +95,10 @@ sub load_cuttext {
         cut_retrieve        => $cutid,
     };
 
-    LJ::CleanHTML::clean_event( \$text, $cleanhtml_opts );
+    #load and prepare text of entry
+    my $text = LJ::CleanHTML::quote_html( $entry_obj->event_html($cleanhtml_opts), $get->{nohtml} );
+
+    # LJ::item_toutf8( $journal, \$subject, \$text ) if $entry_obj->props->{unknown8bit};
```

Comment:
I'm concerned about this guy, because I really have no idea what's up with it. Is there anything in our db that might be in weird 8-bit encodings? Is this safe to re-enable if we're not using "event_raw"? 

The reason I suspected it's not necessary is because LJ/S2/EntryPage.pm doesn't do any of this.